requirements.txt

```diff
@@ -8,3 +8,69 @@ pytest==7.4.1
 python-dotenv==0.21.1
 pytz==2022.7.1
 taskipy==1.12.0
+aiohttp==3.8.6
+aiosignal==1.3.1
+async-timeout==4.0.3
+asyncpg==0.27.0
+attrs==23.1.0
+black==23.7.0
+build==0.10.0
+CacheControl==0.13.1
+certifi==2023.7.22
+cffi==1.16.0
+charset-normalizer==3.3.0
+cleo==2.0.1
+click==8.1.7
+colorama==0.4.6
+coverage==7.3.2
+crashtest==0.4.1
+cryptography==41.0.4
+discord.py==2.3.2
+distlib==0.3.7
+dulwich==0.21.6
+filelock==3.12.4
+frozenlist==1.4.0
+idna==3.4
+importlib-metadata==6.8.0
+iniconfig==2.0.0
+installer==0.7.0
+isort==5.12.0
+jaraco.classes==3.3.0
+jeepney==0.8.0
+jsonschema==4.17.3
+keyring==24.2.0
+more-itertools==10.1.0
+msgpack==1.0.7
+multidict==6.0.4
+mypy-extensions==1.0.0
+packaging==23.2
+pathspec==0.11.2
+pexpect==4.8.0
+pkginfo==1.9.6
+platformdirs==3.11.0
+pluggy==1.3.0
+poetry==1.6.1
+poetry-core==1.7.0
+poetry-plugin-export==1.5.0
+psutil==5.9.6
+ptyprocess==0.7.0
+pycparser==2.21
+pyproject_hooks==1.0.0
+pyrsistent==0.19.3
+pytest==7.4.1
+pytest-cov==4.1.0
+python-dotenv==0.21.1
+pytz==2022.7.1
+rapidfuzz==2.15.2
+requests==2.31.0
+requests-toolbelt==1.0.0
+SecretStorage==3.3.3
+shellingham==1.5.3
+taskipy==1.12.0
+tomli==2.0.1
+tomlkit==0.12.1
+trove-classifiers==2023.9.19
+urllib3==2.0.6
+virtualenv==20.24.5
+yarl==1.9.2
+zipp==3.17.0
```

Comment:
It won't be good if this file is filled with chain dependencies. It will be better to just include the main library. For example, `pytest-cov` requires `pytest` and `coverage` lib, so just write the main lib manually by providing the required version
```
pytest-cov^=4.1.0 
```
or
```
pytest-cov==4.1.0 
```